src/tests/ftest/util/dfuse_utils.py

```diff
@@ -423,7 +423,7 @@ def start_dfuse(test, dfuse, pool=None, container=None, **params):
     if pool:
         params['pool'] = pool.identifier
     if container:
-        params['cont'] = container.uuid
+        params['cont'] = container.identifier
```

Comment:
Thanks. FWIW, I still have work in progress to update 99% of ftest to use labels

Reply:
Luckily I have a PR to make this same change and there wasn't fallout.